src/components/LoginPanel.tsx

```diff
@@ -8,26 +8,26 @@ import React, { useEffect } from "react";
 import styled, { useTheme } from "styled-components";
 import Title from "@/components/Title/Title";
 import { NavBarHeight } from "@/components/NavBar/NavigationBar";
+import Paper from "@mui/material/Paper";
 
 export const LoginMain = styled.main`
-    height: calc(100vh - ${NavBarHeight});
+    height: calc(100vh - ${NavBarHeight} * 2);
```

Comment:
what does the x2 do? where is the panel positioned relative to the page? can this be based on % (of the div that doesn't include the nav bar) rather than vh?

Reply:
As discussed with you and @Var1377, the height is expected to be height: calc(100vh - ${NavBarHeight}); but to center the component we need to have the height as the whole view port minus 2 nav bar heights hence the *2.